nodeadm/internal/api/types.go

```diff
@@ -122,4 +122,10 @@ type Feature string
 const (
 	// InstanceIdNodeName will use EC2 instance ID as node name
 	InstanceIdNodeName Feature = "InstanceIdNodeName"
+
+	// AggressiveImagePullTradeoff enables a parallel image pull for container
+	// images. This will use more instance CPU and Memory during image pull, but
+	// may result in faster image pull times. This flag will be ignored on
+	// instances with memory and vCPU below a certain threshold.
```

Comment:
for future ref, this `internal/api` package is sealed so no need to replicate the doc comments

Reply:
thanks, removed.